core/src/test/codegen/config.fmpp

```diff
@@ -390,11 +391,16 @@ data: {
     ]
 
     # List of methods for parsing extensions to "CREATE [OR REPLACE]" calls.
-    # Each must accept arguments "(SqlParserPos pos, boolean replace)".
+    # Each must accept arguments "(SqlParserPos pos, SqlCreateOrReplace replace)".
```

Comment:
nit: wrap at 80 characters

Reply:
The line is 83 characters, and the text makes sense to group together. Is it really worth it to split the line over 3 extra characters? I feel like that might hurt readability more than it would help.

Reply:
80 characters is a pretty widely-held standard. Even if it may make a line slightly less readable, it's a good idea to stick to that standard in practice. In this case, you could break the line after "arguments" since the remaining portion in quotes is kind of a single unit.

Reply:
Ok, will do.